src/jit/lsra.h

```diff
@@ -1131,7 +1131,7 @@ class LinearScan : public LinearScanInterface
     int compareBlocksForSequencing(BasicBlock* block1, BasicBlock* block2, bool useBlockWeights);
     BasicBlockList* blockSequenceWorkList;
     bool            blockSequencingDone;
-    void addToBlockSequenceWorkList(BlockSet sequencedBlockSet, BasicBlock* block);
+    void addToBlockSequenceWorkList(BlockSet sequencedBlockSet, BasicBlock* block, BlockSet predSet);
     void removeFromBlockSequenceWorkList(BasicBlockList* listNode, BasicBlockList* prevNode);
```

Comment:
A value arg should be fine here, since the bitset is cleared upon each call to `addBlockSequenceWorklist`. This should probably use `BlockSet_ValArg_T`, though.